reports/Treat2015Visulize.Rmd

```diff
@@ -0,0 +1,262 @@
+location.df <- unique(Treat$field[, c('lat', 'long', 'observation_date', 
+                                       'state', 'country')])
+location.df$year <- location.df$observation_date
+
+location.df$obsDate <- ymd(paste(location.df$year, 1, 1, sep="-"))
+location.df <- location.df[order(location.df$lat, location.df$long, location.df$obsDate),]
+location.df$yrCut <- cut(location.df$year, c(1950, 1960, 1970, 1980, 1990, 2000, 2013), dig.lab=4)
+#location.df$yrCut <- location.df$year
+location.df$latCut <- cut(location.df$lat, seq(-90, 90, by=0.05))
+location.df$longCut <- cut(location.df$long, seq(-180, 180, by=0.05))
+
+timeSpaceCounts <- location.df %>%
+  mutate(inUSA = country %in% 'United States') %>%
+  group_by(yrCut, latCut, longCut, inUSA) %>%
+  tally
+```
+```{r plotTime}
+ggplot(location.df, aes(x=obsDate)) + geom_histogram()
+
+mapWorld <- borders("world", colour="gray80", fill="gray80") # create a layer of borders
+#ggplot() + mapWorld
+ggplot(location.df) +
+  mapWorld + 
+  #geom_hex(aes(x=long, y=lat), bins=200) + 
+  geom_point(aes(x=long, y=lat), alpha=0.1, color='red') +
+  scale_fill_gradient(trans='log10') +
+  theme_bw() +
+  theme(text=element_text(size=18)) +
+  labs(x='', y='') +
+  ylim(45, 90) + 
+  coord_map(projection='azequidistant') +
+  facet_wrap(~yrCut)
+
+#ggplot(location.df, aes(x=long, y=lat)) + geom_point(alpha=0.1)
+
+ggplot(timeSpaceCounts, aes(x=n)) + geom_histogram() + 
+  scale_x_log10() + scale_y_log10()+ facet_wrap(~inUSA, scales='free') +
+  labs(title='Close space-time sites by USA status')
+```
+
+# Site locations
+## Lat-lon map
+```{r mapSites}
+
+mapWorld <- borders("world", colour="gray80", fill="gray80") # create a layer of borders
+#ggplot() + mapWorld
+ggplot(unique(Treat$field[, c('lat', 'long')])) +
+  mapWorld + 
+  geom_hex(aes(x=long, y=lat), bins=200) + 
+  scale_fill_gradient(trans='log10') +
+  theme_bw() +
+  theme(text=element_text(size=18)) +
+  labs(x='', y='')
+```
+
+# Measruements distribution
+
+```{r measurementHist}
+ggplot(subset(Treat$sample, measurementID %in% c('14c_age', 'n_tot', 'c_tot', 'loi')), aes(x=value)) +
+  geom_histogram() +
+  facet_wrap(measurementID~unit, scales='free')
+
+ggplot(subset(Treat$sample, !measurementID %in% c('14c_age', 'n_tot')), aes(x=value)) +
+  geom_histogram() +
+  scale_x_log10() +
+  facet_wrap(measurementID~unit, scales='free')
+```
+
+# Comparison with ISCN3
+
+```{r loadISCN3, cache=TRUE}
+ISCN <- processData_ISCN3(dir='../repoData/ISCN_3/')
+```
+
+```{r trimISCN}
+#only keep things that overlap
+ISCN$measurement <- subset(ISCN$measurement, as.character(type) %in% c(as.character(Treat$measurement$type), '14c'))
+
+ISCN$sample <- subset(ISCN$sample, as.character(measurementID) %in% as.character(ISCN$measurement$measurementID))
+
+ISCN$field <- subset(ISCN$field, as.character(fieldID) %in% as.character(ISCN$sample$fieldID))
+
+ISCN$field <- ISCN$field %>%
+  separate(observation_date, c('monthStr', 'dayStr', 'yearStr'), 
```

Comment:
We've already seen this (117-125) logic above, right? That suggests it should be abstracted away into a function.